src/frontend/src/storage/index.ts

```diff
@@ -156,6 +180,32 @@ const migrated = (): Anchors | undefined => {
   return anchors;
 };
 
+/** "Low-level" functions to read anchors from and write anchors to IndexedDB */
+
+const readIndexedDB = async (): Promise<Anchors | undefined> => {
+  const item: unknown = await idb.get("anchors");
+
+  if (item === undefined) {
```

Comment:
isNullish or can never be null or null is a value?

Reply:
I've avoided `isNullish` because if the value is `null` then (AFAICT) it was actually set to `null` in the DB (as opposed to simply not being set). I think it makes sense to check for `undefined` explicitly but don't feel too strongly about it.

Reply:
Not knowing the code base I wasn't unsure if it was expected or not therefore the question but sure makes sense.